src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services1-2.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
```

Comment:
I am not sure. It's just that docblocks seem more structured to me than simple comments. Should I change into simple comments?